deployments/helm-chart/templates/controller-leader-election-configmap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "nginx-ingress.leaderElectionName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "nginx-ingress.labels" . | nindent 4 }}
+data:
```

Comment:
Do you need the `data` field if is empty?

Reply:
Nope 😄